Library/Homebrew/manpages.rb

```diff
@@ -14,6 +14,21 @@ module Homebrew
   module Manpages
     extend T::Sig
 
+    Variables = Struct.new(
```

Comment:
Could/should this just be a `Hash` instead?

Reply:
I don't think so, though I don't have a lot of experience with ERB generation. This gives us the benefit of erroring if any of the properties are fat-fingered below.

Reply:
Ok, thanks!